Shared/ViewModels/ActiveSessionsViewModel.swift

```diff
@@ -0,0 +1,138 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2024 Jellyfin & Jellyfin Contributors
+//
+
+import Combine
+import Foundation
+import JellyfinAPI
+import OrderedCollections
+import SwiftUI
+
+final class ActiveSessionsViewModel: ViewModel, Stateful {
+
+    // MARK: Action
+
+    enum Action: Equatable {
+        case error(JellyfinAPIError)
+        case loadSessions
+        case refreshSessions
+    }
+
+    // MARK: State
+
+    enum State: Hashable {
+        case initial
+        case loading
+        case loaded
+        case refreshing
```

Comment:
I'm going to have to think about having `loading` and `refreshing`. Instead, we should probably have the passive-timer loading sessions state be through a `BackgroundState` as well as a hard refresh state.

See `HomeViewModel` for a similar implementation.